zip-0311.rst

```diff
@@ -2,7 +2,262 @@
+- A third party needs to verify that a payment between sender and recipient was executed
+  successfully.
+
+When a transaction involves only transparent addresses, the sender and recipient can verify payment by examining the blockchain.  A third party can also perform this verification if they know the transparent addresses of the involved parties.
+
+However, if the transaction involves shielded addresses, the blockchain by itself does not contain enough information to allow a record of the payment to be reconstructed and verified.
+
+Let us examine the types of transaction that might occur and when a method of payment disclosure would be useful:
+
+transparent --> transparent
+  The source, destination and amount are visible on the public blockchain. Payment
+  disclosure is not required to view the transaction.
+
+transparent --> shielded
+  The destination address and amount sent to that address cannot be confirmed. Payment
+  disclosure is required.
+
+shielded --> transparent
+  The recipient can see the amount received at their destination address, but cannot
+  identify the sender. Requires either an interactive payment disclosure, or a
+  non-interactive payment disclosure that reveals the sender's address.
+
+shielded --> shielded
+  The sender, recipient and amount are unknown. Payment disclosure required.
+
+
+Use cases
+=========
+
+Managing contributions for an event
+-----------------------------------
+Alice runs an event on behalf of Bob and Carol, who both agree to split the cost equally.
+Alice receives a single payment for half the amount, and wants proof of who it came from
+(so she knows which person to follow up with for the remaining amount). Carol can provide
+a payment disclosure that reveals:
+
+- The correct amount was sent to the Alice's recipient address in the given transaction.
+- Carol was the sender of that transaction (more precisely, Carol controls the spend
+  authority used in that transaction).
+
+and does not reveal:
+
+- Carol's payment address.
+- Any of Carol's other diversified payment addresses.
+- Any of Carol's other transactions.
+
+Donation drive
+--------------
+
+Diana is a well-known individual with a public Sapling payment address (for receiving
+donations). She runs a matching donation drive to the non-profit Totally Legit Bunnies,
+and wants to prove to her followers that she correctly matched their dontations. Diana
+can create a payment disclosure, and post it to her social media account, that reveals:
+
+- The correct amount was sent to the Totally Legit Bunnies' recipient address in the
+  given transaction.
+- The sender of the transaction is the payment address that is associated with Diana.
+
+and does not reveal:
+
+- Any of Diana's other diversified payment addresses.
+- Any of Diana's other transactions.
+
+Merchant dispute
+----------------
+
+Edward goes to CoinCafe to buy a sandwich, and sends funds from his Sapling address to
```

Comment:
I'm not sure that the sandwich example is very compelling :-) Maybe the tx is for a higher-value item, such as a car from a second-hand car dealer?